web_src/fomantic/build/semantic.css

```diff
@@ -3689,6 +3689,11 @@ body.dimmable > .dimmer {
   will-change: transform, opacity;
 }
 
+.ui.dropdown .menu.short {
+  max-width: 60vh;
+  overflow-y: auto;
+}
+
```

Comment:
this can be moved to base.css

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/24212/commits/00fb639246c795673bb1bdaa95858827316504be